x-pack/plugins/observability/server/plugin.ts

```diff
@@ -14,23 +20,76 @@ import {
   AnnotationsAPI,
 } from './lib/annotations/bootstrap_annotations';
 import type { RuleRegistryPluginSetupContract } from '../../rule_registry/server';
+import { PluginSetupContract as FeaturesSetup } from '../../features/server';
 import { uiSettings } from './ui_settings';
 import { registerRoutes } from './routes/register_routes';
 import { getGlobalObservabilityServerRouteRepository } from './routes/get_global_observability_server_route_repository';
 
 export type ObservabilityPluginSetup = ReturnType<ObservabilityPlugin['setup']>;
 
+interface PluginSetup {
+  features: FeaturesSetup;
+  ruleRegistry: RuleRegistryPluginSetupContract;
+}
+
+const OBS_CASES_ID = 'observabilityCases';
```

Comment:
I am confused with the naming of the variables. `OBS_CASES_ID` is for the server and the `CASES_APP_ID` is for the UI, right? If so, I think is best to rename `OBS_CASES_ID` to `SERVER_CASES_APP_ID`.